src/Sylius/Bundle/CoreBundle/Doctrine/ORM/ProductRepository.php

```diff
@@ -207,6 +207,23 @@ public function findOneByIdAndChannel($id, ChannelInterface $channel = null)
     /**
      * {@inheritdoc}
      */
+    public function findByTaxonCodeAndChannel($code, ChannelInterface $channel)
```

Comment:
A product is diabled permanently? If not, then why customer cannot see product avaiable for them but not exactly in this moment?